### PR TITLE
New package: Friction.Friction version 0.9.4

### LINKED ISSUE #119024: [Package Request]: friction
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://friction.graphics/releases/friction-094.html
Publisher: friction2d
Package Name: friction
Description: open-source, vector-based application designed for creating web animations and videos.
Package Version: 0.9.4
Installer URL: https://github.com/friction2d/friction/releases/download/v0.9.4/friction-0.9.4-setup-win64.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## What's changed

- Resolve #119024
- Related https://github.com/friction2d/friction/issues/76

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122156)

### CODE REVIEW

**Comment by Dragon1573** [comment]
## Claim

This package CAN be installed silently and unattededly. Please re-run the Azure Pipelines and verify if there are difference between pipelines and locally verification.

## Details

```text
PS (local)> .\Tools\SandboxTest.ps1 .\manifests\f\Friction\Friction\0.9.4\
--> Validating Manifest
清单验证成功。

--> Checking dependencies
--> Starting Windows Sandbox, and:
    - Mounting the following directories:
      - C:\Users\Dragon1573\AppData\Local\Temp\SandboxTest as read-only
      - D:\Repository\winget-pkgs as read-and-write
    - Installing WinGet
    - Configuring Winget
    - Installing the Manifest 0.9.4
    - Refreshing environment variables
    - Comparing ARP Entries
```

```text
--> Installing WinGet
--> Disabling safety warning when running installer
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Enabled admin setting 'LocalManifestFiles'.
Enabled admin setting 'LocalArchiveMalwareScanOverride'.


--> Installing the Manifest 0.9.4

Found Friction [Friction.Friction] Version 0.9.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/friction2d/friction/releases/download/v0.9.4/friction-0.9.4-setup-win64.exe
  ██████████████████████████████  25.0 MB / 25.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName            DisplayVersion Publisher ProductCode                                Scope
-----------            -------------- --------- -----------                                -----
Friction version 0.9.4 0.9.4          Friction  {12D25577-88FD-4C5A-AF69-CCA597CAC44E}_is1 User
```

image

File: manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - DisplayName: Friction version 0.9.4
+    Publisher: Friction
+    DisplayVersion: 0.9.4
+    ProductCode: '{12D25577-88FD-4C5A-AF69-CCA597CAC44E}_is1'
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/friction2d/friction/releases/download/v0.9.4/friction-0.9.4-setup-win64.exe
+  InstallerSha256: 63D84A44B2BE5A9921720D782FCD5C7294863FB4BBD54BADE441E0A18C0055B9
+  InstallerSwitches:
+    Custom: /ALLUSER
```
Comment (Trenly) [inline manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml:24]:
```suggestion
    Custom: /ALLUSERS
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  AppsAndFeaturesEntries:
+  - DisplayName: Friction version 0.9.4
+    Publisher: Friction
+    DisplayVersion: 0.9.4
```
Comment (Trenly) [inline manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml:28]:
```suggestion
```
DisplayVersion should be left off if it's the same as PackageVersion

File: manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/friction2d/friction/releases/download/v0.9.4/friction-0.9.4-setup-win64.exe
+  InstallerSha256: 63D84A44B2BE5A9921720D782FCD5C7294863FB4BBD54BADE441E0A18C0055B9
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+  AppsAndFeaturesEntries:
+  - DisplayName: Friction version 0.9.4
+    Publisher: Friction
+    DisplayVersion: 0.9.4
```
Comment (Trenly) [inline manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml:17]:
```suggestion
```

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Friction.Friction
+PackageVersion: 0.9.4
+InstallerType: inno
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/friction2d/friction/releases/download/v0.9.4/friction-0.9.4-setup-win64.exe
+  InstallerSha256: 63D84A44B2BE5A9921720D782FCD5C7294863FB4BBD54BADE441E0A18C0055B9
```
Comment (stephengillie) [inline manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml:11]:
```suggestion
  InstallerSha256: 63D84A44B2BE5A9921720D782FCD5C7294863FB4BBD54BADE441E0A18C0055B9
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

File: manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Friction.Friction
+PackageVersion: 0.9.4
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/f/Friction/Friction/0.9.4/Friction.Friction.installer.yaml:6]:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
With the dependency added, it encounters a pipeline limitation:
image

- https://github.com/microsoft/winget-pkgs/issues/107639

**Comment by Dragon1573** [comment]
Sorry, I can't get it...

 Does it means this package requires a new feature of WinGet?  The feature isn't available now, so it blocks the Azure Pilepines?

**Comment by stephengillie** [comment]
> Sorry, I can't get it...
> 
> Does it means this package requires a new feature of WinGet? The feature isn't available now, so it blocks the Azure Pilepines?

Sorry for being unclear. This is a limitation of the current pipelines, not with your package.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**